packages/ripple-nuxt-tide/modules/landing-page/pages/index.vue

```diff
@@ -38,6 +43,24 @@ export default {
     anchorLinks: Array,
     sidebar: Boolean
   },
+  computed: {
+    topicsAndTags () {
+      if (this.page.field_show_topic_term_and_tags) {
+        let metaTags = []
+        if (this.page.field_topic) {
+          metaTags.push(this.page.field_topic)
+        }
+        if (this.page.field_tags) {
+          metaTags = metaTags.concat(this.page.field_tags)
+        }
+        return metaTags.map(tag => ({
+          text: tag.name,
+          url: (tag.path && tag.path.alias) ? tag.path.alias : ''
```

Comment:
Not sure what tag url Tide will pass to us. If it's a Nuxt route path instead of CMS path, I think we will need some prefix e.g `/tag/` so we can handle it in Nuxt. @alan-cole @dylankelly

Reply:
I had discussed with Sonny before implementing. He said it would be the full path including the prefix.

Reply:
I have tested this with https://nginx-php-content-vic-pr-649.lagoon.vicsdp.amazee.io (SDPA-2987). It works as expected.